### ISSUE TITLE
Unable to create repositories

### ISSUE THREAD

**Issue by klenin** [icpc/start-here#1]
I wanted to explore the possibility of publishing our problems here,
but it seems that I am unable to create new repositories inside of `icpc` organization.
For reference, I have published a single problem (from the last year, so it is not secret)
under my own account:
https://github.com/klenin/demo-problem

**Comment by niemela** [comment]
@klenin What region and year do you want to create a repository for?

**Comment by klenin** [comment]
The example I provided was taken from Far Eastern Subregional of NEERC, 2017.
However, my understanding was that this project is a mirror of all past (and even some future)
problems, so we are speaking about >100 repositories per region.
Or did I misunderstood the intent of `icpc` organization?

**Comment by niemela** [comment]
The intent is to provide private repositories for developing problems to future contest, and private or public repositories for past contest, depending on wether you want to share the problems with the public or only selectively.

This would make 1 repository per region per year. So even if everybody signs up (which will take some time) and they are all private (which I doubt they will be) we would have repositories for more than 100 years. That said, it might very well make sense to collect all of the (ever growing) past in a single repository per region. At least if the region wants to share all their problems with the public.

There is a [ne-fareast](https://github.com/icpc/ne-fareast/) repository, and as fas as I can see you're in the right team to have access to it. Is it not working correctly for you?

**Comment by klenin** [comment]
You suggest repository per contest. We (in our region, and some other regions of NE at least) currently have repository per problem.
So number of repositories is measured in hundreds.

**Comment by klenin** [comment]
I guess we could just publish some simple annotations, like problem names and links to our system...

**Comment by wmacevoy** [comment]
I don’t think we can sustain that model.  Why not have them in one repository?  They are used and published as a set...

**Comment by klenin** [comment]
Sometimes so, however:
1) Problems have different authors, possibly with different access privileges
  (not in this org, but certainly in actual contest systems)
2) Submitting problem for consideration does not automatically mean access to all other problems.
3) Contest problem set is created by selecting from a pool of available problems. Non-selected problems should remain in the pool and may be used in future problem sets.
4) Training problem sets are often created by mixing problems from different contests
5) At least some contest systems which support problem development expect repo-per-problem and actual development is done this way

That said, it is certainly possible to create and publish octopus merges from all contest and problem repositories. Git is wonderful :)
The downside is that such repo will be read-only, so it is only marginally better than just publishing zip file with a problem package, which everybody does already.

**Comment by wmacevoy** [comment]
Can old problems move into a single repository - "yardsale" or "attic" ?  Say after 1 or 2 years?  This would keep the total active private repositories to a reasonable minimum.

**Comment by klenin** [comment]
That is basically what I suggested above.
Note that merging multiple independent repositories will produce "interesting" git histories.
Unfortunately it seems that is the only possible solution.
I will try to implement such exporter, but it is not as easy as just pushing existing repositories to another remote.
Perhaps sometime in the fall.

**Comment by wmacevoy** [comment]
Ok we’d like to try this and see how many repos these burn
